src/euphorie/client/browser/choice.py

```diff
@@ -0,0 +1,107 @@
+from euphorie.client import utils
+from euphorie.client.navigation import getTreeData
+from plone import api
+from plone.memoize.instance import memoize
+from Products.Five import BrowserView
+
+
+class IdentificationView(BrowserView):
+    """A view for displaying a choice in the identification phase."""
+
+    variation_class = "variation-risk-assessment"
+
+    @property
+    @memoize
+    def webhelpers(self):
+        return api.content.get_view("webhelpers", self.context.aq_parent, self.request)
+
+    def check_render_condition(self):
+        # Render the page only if the user can inspection rights,
+        # otherwise redirect to the start page of the session.
+        if not self.webhelpers.can_inspect_session:
+            return self.request.response.redirect(
+                "{session_url}/@@start".format(
+                    session_url=self.webhelpers.traversed_session.absolute_url()
+                )
+            )
+        if self.webhelpers.redirectOnSurveyUpdate():
+            return
```

Comment:
I do not understand this if clause.
It seems to me the code is equivalent to:
```suggestion
        self.webhelpers.redirectOnSurveyUpdate():
```
To me

Reply:
Copied from https://github.com/euphorie/Euphorie/blob/781195c2ee117795f3c1d216f94398968a81728b/src/euphorie/client/browser/risk.py#L627

My guess is that it's to make sure that any code that might be added after this line is not executed if the redirect kicks in.